tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -553,6 +553,21 @@ public static class CIVisibility
             /// Configuration key for set the test session name
             /// </summary>
             public const string TestSessionName = "DD_TEST_SESSION_NAME";
+
+            /// <summary>
+            /// Configuration key for a kill-switch that allows to explicitly disable retries even if the remote setting is enabled.
+            /// </summary>
+            public const string FlakyRetryEnabled = "DD_CIVISIBILITY_FLAKY_RETRY_ENABLED";
```

Comment:
Don't forget to add these to the allow list in dd-go

Reply:
Already merged 😎 https://github.com/DataDog/dd-go/pull/151178/files